fasthtml/cli.py

```diff
@@ -51,3 +52,22 @@ def railway_deploy(
     railway_link.__wrapped__()
     if mount: _run(f"railway volume add -m /app/data".split())
     _run(f"railway up -c".split())
+
+# %% ../nbs/api/09_cli.ipynb
+@call_parse
+def project_init():
+    """
+    initialise a new FastHTML project.
+    for now, we just create .gitignore file.
+    """
+    print("running fh_init...")
+    local_gitignore = Path.cwd() / ".gitignore"
+    remote_gitignore = "https://raw.githubusercontent.com/answerdotai/fasthtml/main/.gitignore"
```

Comment:
This is just a placeholder URL for now. If we decide this change is worth merging, I'll add a dedicated `gitignore.example` file to the repo.